tests/test_intertidal.py

```diff
@@ -54,7 +54,7 @@ def test_intertidal_cli():
             "--output_version",
             "0.0.1",
             "--tide_model",
-            "FES2014",
+            "EOT20",

```

Comment:
```suggestion
            "FES2014",

```

This is the only place where we actually do need to set "FES2014": for consistency because that's what our historical tests ran on, and because we don't actually have "EOT20" data available to our tests right now (because it's coming from an old Dropbox link). Both things to improve in the future, but for now "FES2014" is OK 🙂